src/ui/server/api/keystone/application/index.ts

```diff
@@ -200,6 +201,36 @@ const application = {
         throw new Error('Updating application business');
       }
     },
+    businessContact: async (id: string, update: object) => {
+      try {
+        console.info('Updating application business contact');
+
+        const variables = {
+          where: { id },
+          data: update,
+        };
+
+        const response = (await apollo('POST', updateBusinessContactMutation, variables)) as ApolloResponse;
+
+        if (response.errors) {
+          console.error('GraphQL error updating application business ', response.errors);
+        }
+
+        if (response?.networkError?.result?.errors) {
+          console.error('GraphQL network error updating application business ', response.networkError.result.errors);
+        }
+
+        if (response?.data?.updateBusinessContactDetail) {
+          return response.data.updateBusinessContactDetail;
+        }
+
+        console.error(response);
+        throw new Error('Updating application business');
+      } catch (err) {
+        console.error(err);
+        throw new Error('Updating application business');
+      }
+    },
```

Comment:
Not a blocker can be done afterwards, but the logs need to match - the first was says `business contact` but the others just say `business`, would be confusing if debugging